python/ray/includes/ray_config.pxi

```diff
@@ -119,7 +122,8 @@ cdef class Config:
 
     @staticmethod
     def node_manager_forward_task_retry_timeout_milliseconds():
-        return RayConfig.instance().node_manager_forward_task_retry_timeout_milliseconds()
+        return RayConfig.instance()\
```

Comment:
don't use `\`, instead do something like

```
return (RayConfig.instance()
        .node_manager_forward_task_retry_timeout_milliseconds())
```

this applies in other places as well.

Reply:
I fixed it for the ones in this file, for some cython constructs we do need to use \ (this is quite standard, see https://github.com/apache/arrow/blob/master/python/pyarrow/_cuda.pyx)